src/components/Inputs/CheckboxInput/CheckboxInput.js

```diff
@@ -80,7 +81,7 @@ export const CheckboxInput = ({
           />
           <Facade />
         </div>
-        <Body.Regular400>{children}</Body.Regular400>
+        <Body.Regular400 color={COLORS.GRAY_PRIMARY}>{children}</Body.Regular400>
```

Comment:
This kind of thing (here and in CMS/1991) is a side effect of making the color prop optional, which I did because it seems easier to do the CMS updates that way.